### DISCUSSION TITLE
Errors when sending emails

### DISCUSSION THREAD

**Discussion by Toni202012569** [postalserver/postal#2671] [Help with using Postal]
Please tell me what is the problem? Why is there no email data and what errors am I getting? How to solve this? Thank you in advance.
![Скриншот 23-10-2023 22 09 11](https://github.com/postalserver/postal/assets/139015871/5bb05379-f366-4c84-a3bc-a78c9b646e0b)

**Comment by willpower232** [comment]
You would need to go into the messages and see what the error was on the right hand side.

**Comment by Toni202012569** [comment]
Yes thank you. I realized what the problem is, I don’t have it anymore
